CMakeLists.txt

```diff
@@ -86,6 +86,10 @@ if(NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
     "MinSizeRel" "RelWithDebInfo")
 endif()
 
```

Comment:
CMake provides a find_package() mechansim for including external dependancies.  The typical scenerio I have encountered is
1) download, compile, install package (tbb in this case)
2) from the many tools that depnd on TBB, use find_package to point to that one build of tbb


Then from independant build add the following to it's CMakeList.txt

find_package(tbb REQUIRED)
add_executable(test_tbb test.cpp)
target_link_libraries(test_tbb PUBLIC tbb::tbb )